src/renderer/components/deposit/add/SymDeposit.tsx

```diff
@@ -276,6 +259,39 @@ export const SymDeposit: React.FC<Props> = (props) => {
     [depositFeesRD, zeroDepositFees]
   )
 
+  const oThorchainFees = useMemo(() => Helper.getThorchainFees(depositFeesRD), [depositFeesRD])
+  const thorchainFees: DepositFees = useMemo(
+    () =>
+      FP.pipe(
+        oThorchainFees,
```

Comment:
[nitpick] `oThorchainFees` -> `Helper.getThorchainFees(depositFeesRD)`. No need to have another `useMemo` for `oThorchainFees` then.